ups/eupspkg.cfg.sh

```diff
@@ -15,6 +15,15 @@ config()
 
         # Disable unicode regex support to make the binary more portable
 	./bootstrap.sh --without-libraries=mpi --without-icu --prefix="$PREFIX" $WITH_TOOLSET
+
+    # On Python 3 boost can not find the include directory if it uses an architecture
+    # specifier such as 3.5m (PEP-3149). We therefore have to modify the config jam file
+    # to add the include file as determined by asking python.
+    PYTHON_VERSION=$(python -c 'import sys; print("%d.%d" % (sys.version_info[0], sys.version_info[1]))')
+    if [ $(bc <<< "$PYTHON_VERSION > 3.0") -eq 1 ]; then
```

Comment:
Since you're picking what to print from Python, I suggest you just take `sys.version_info.major` and compare with `3` in the shell, without using `bc`.  This will avoid potential problems with double-digit minor versions.  You could also use sys.hexversion, but that's more obscure.  (The test as-is might have problems if someone had a Python 3.0 or 3.0.1 installation, but those people are likely to have other problems, too.)